tests/DocBlock/TypeExpressionTest.php

```diff
@@ -146,6 +146,8 @@ public static function provideGetTypesCases(): iterable
         yield ['\\Closure(string|int, bool): bool', ['\\Closure(string|int, bool): bool']];
 
         yield ['array  <  int   , callable  (  string  )  :   bool  >', ['array  <  int   , callable  (  string  )  :   bool  >']];
+
+        yield ['\'a\\\'s"\\\\\n\r\t\'|"b\\"s\'\\\\\n\r\t"', ['\'a\\\'s"\\\\\n\r\t\'', '"b\\"s\'\\\\\n\r\t"']];
```

Comment:
Can we have a few more? What is the smallest accepted type? `'\''`?

Reply:
added smallest type - `''` empty string and a few other simple strin before the complex test